Sources/SentryCrash/Recording/Tools/SentryCrashSysCtl.c

```diff
@@ -190,6 +190,24 @@ sentrycrashsysctl_timevalForName(const char *const name)
     return value;
 }
 
+struct timeval
+sentrycrashsysctl_currentProcessStartTime()
+{
+    size_t len = 4;
+    int mib[len];
+    struct kinfo_proc kp;
+
+    sysctlnametomib("kern.proc.pid", mib, &len);
+    mib[3] = getpid();
+    len = sizeof(kp);
+
+    if (0 != sysctl(mib, 4, &kp, &len, NULL, 0)) {
+        SentryCrashLOG_ERROR("Could not get current process start time: %s", strerror(errno));
```

Comment:
if this somehow fails and we don't have such an app start up, can we fallback to the `uiapplicationdelegate` notifications?

Reply:
We use anyway the UIApplicationDelege. Do you mean maybe we fall back to setting the app start in `load` as we had it before? We already use similar `sysctl` functions here, so I would expect it to work. If it doesn't we can always do a fix.